src/js/main.js

```diff
@@ -296,7 +297,47 @@ Site.Modules = {
     _this.$videos.on('mouseleave', function(event) {
       this.pause();
     });
-  }
+  },
+
+  bindScroll: function() {
+    var _this = this;
+
+    // On window scroll
+    $(window).scroll(function(event) {
+      // Get the scroll position
+      var scrollTop = $(window).scrollTop();
+
+      // ...handle it
+      _this.handleScroll(scrollTop);
+    });
+  },
+
+  handleScroll: function(scrollTop) {
+    var _this = this;
+
+    if(!_this.played && scrollTop > _this.threshold) { // First video is to play and postition is below threshold
+      _this.playAnimation();
+    }
+  },
+
+  playAnimation: function() {
+    var _this = this;
+
+    _this.played = true;
```

Comment:
I think this is a bit of an issue too. Maybe fine when playing fully once. Otherwise weird is doesnt retrigger

Reply:
What do you mean an issue? The animation should never retrigger.

Reply:
Why should it never retrigger? The intention is just for these to play the first time down on the first scroll and then not again?

Reply:
yeah